server/controllers/promise.js

```diff
@@ -0,0 +1,124 @@
+import Aid from  "../models/aid"
+import Report from "../models/report"
+
+class AidDb {
+    static find(param) {
+      return new Promise((resolve, reject) => {
```

Comment:
you do not need to wrap these functions in a promise, they are already promises.
```
return Aid.find(param);
```
is all you need to do